content/24.vaccines-novel.md

```diff
@@ -200,31 +190,33 @@ These vaccines therefore represent implementations of a new platform technology.
-Therefore, vaccine efficacy data from a large study population is not yet available.
-
-**Real-World Safety and Efficacy:**
-The efficacy of ZyCoV-D is estimated to be 66.6% [@doi:10/hvqb].
-Because phase III data is not yet available for INO-4800, the VE is not yet know.
-
-Studies have examined the ability of INO-4800 to induce an immune response that can neutralize existing VOC.
-They assessed neutralization of several VOC relative to the index strain [@doi:10.1038/s41541-021-00384-7].
-They found no difference in neutralization between the index strain and the Gamma VOC (P.1), but neutralization of the Alpha and Beta VOC was significantly lower (approximately two and seven times, respectively) [@doi:10.1038/s41541-021-00384-7].
-These findings are in line with the shifts in efficacy reported for other vaccines [@individual-vaccines-traditional].
+##### Trial Safety and Immunogenicity
+
+The INO-4800 trials began with a phase I trial evaluating two different doses administered as a two-dose series [@doi:10.1016/j.eclinm.2020.100689].
+This trial found the vaccine to be safe (with only six adverse events (AEs) reported by 39 participants, all grade 1) and effective, with all but three participants of 38 developing serum IgG binding titers to the SARS-CoV-2 S protein [@doi:10.1016/j.eclinm.2020.100689].
+In the phase II trial of 401 adults at high risk of exposure to SARS-CoV-2 similarly supported the safety and efficacy of INO-4800.
+Only one treatment-related AE was observed and the vaccine was found to be associated with a significant increase in neutralizing activity [@doi:10.1101/2021.05.07.21256652]. 
+Results of phase III trials are not yet available [@{url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-U.S.-FDA-Authorization-to-Proceed-with-INNOVATE-Phase-3-Segment-for-its-COVID-19-Vaccine-Candidate-INO-4800-in-the-U.S/default.aspx}; @{url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Further-Expands-INNOVATE-Phase-3-Trial-for-COVID-19-DNA-Vaccine-Candidate-INO-4800-With-Regulatory-Authorization-from-India/default.aspx}; @{url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Expands-INNOVATE-Phase-3-for-INO-4800-its-DNA-Vaccine-Candidate-for-COVID-19-to-include-Colombia-following-Regulatory-Authorization/default.aspx}; @{url:https://ir.inovio.com/news-releases/news-releases-details/2021/INOVIO-Receives-Regulatory-Authorization-to-Conduct-Phase-3-Efficacy-Trial-of-its-COVID-19-DNA-Vaccine-Candidate-INO-4800-in-Mexico/default.aspx}].
+
+Trials of ZyCoV-D have progressed further. 
```

Comment:
Added because it felt too weird to skip this one, given that it was the first DNA vaccine to receive full approval.

Reply:
Agreed